src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/database_tests/OrderlyTests.kt

```diff
@@ -92,6 +92,7 @@ class OrderlyTests : CleanDatabaseTests()
         assertThat(result.parameterValues.keys.count()).isEqualTo(2)
         assertThat(result.parameterValues["p1"]).isEqualTo("param1")
         assertThat(result.parameterValues["p2"]).isEqualTo("param2")
+        assertThat(result.instances).isEqualTo(emptyMap<String, String>())
```

Comment:
This would be a better test if you mocked the `getReportVersion` instances method on mockReportRepo to actually return some instance values, and tested for those.